packages/okta-adapter/src/filters/policy_priority.ts

```diff
@@ -102,6 +102,14 @@ export const createPriorityType = (typeName: string, defaultFieldName: string):
     },
     path: [OKTA, adapterElements.TYPES_PATH, typeName],
   })
+const logDuplicatePriorities = (instances: InstanceElement[]): void => {
+  const duplicatePriorities = _.groupBy(instances, inst => inst.value.priority)
```

Comment:
naming nit + type safety -
```suggestion
  const instanceByPriority = _.groupBy(instances, inst => inst.value?.priority)
```